GameData/RealismOverhaul/RealismOverhaul_Global_Config.cfg

```diff
@@ -331,3 +331,17 @@
 		AllowSignalRelay = False
 	}
 }
+
+//  ==================================================
+//  Remove the KerbNet and Test Subject modules from all parts
+//  ==================================================
```

Comment:
why?
With scansat's RSS compatibility in a questionable state and ROKerbalism not providing scanners of its own, killing kerbnet doesn't seem like a good idea.